app/views/content/blog/my-career-change-to-teaching.md

```diff
@@ -0,0 +1,37 @@
+    
+keywords:
+  - teacher training
+  - trainee teacher course
+  - SCITT
+  - career change
+
+tags:
+  - becoming a teacher
+  - career change to teaching
+---
+
+$ian$
+
+When I was at school I always wanted to be a teacher, after the obvious football career, but when I finished university I didn’t think it was right for me. I had been in education a long time and worried I wasn’t mature enough. I went away and had several different jobs and careers at various levels of responsibility in retail, food and sports, including being self-employed a couple of times. 
+
+Time had passed and I had moved back to Newcastle, met my now wife and was contemplating my future. I was sat having a meal with my wife and she said ‘why don’t you teach?’ My wife is a primary school teacher and she knew that it was what I should be doing with the skill set and experience that my career had given me so far. In particular, I had skills in communication, listening, collaboration, adaptability, empathy and patience, all of which come with maturity and experience.
+
+I took the bull by the horns and looked into the qualification routes that were available, settling on a SCITT as this suited me personally. With a SCITT you are in school from the start and that appealed to my work style and ethic. Being in my late 30s and having financial responsibilities for my family I needed to have support and fortunately I was to study science, which was funded and allowed me to concentrate on my training.
+
+I had made a conscious decision in discussion with my wife that I wouldn’t work at home in the evenings if I could avoid it. I treated my training as if I was going to work every day. I was at the school at 7am and stayed each night until 6pm or later if needed.. This worked brilliantly for me.
+
+For me, the decision to apply to teach later on in life meant I felt even more capable of balancing the demands of the course, dealing with stressful moments, and having successful strategies for coping with competing demands. I knew that having been in stressful job roles and having dealt with difficult situations previously I would be emotionally strong and able to deal with any situation that occurred. I also knew that I was old enough and definitely secure enough in my own character and personality to be open and honest with the students, about both my life and my career to date. 
+
+As far as benefits are concerned, I am paid at a graduate level and have excellent holidays, which I get to spend with my family as we are all off together at the same time. In all it was a great decision and even though my whole training and subsequent teaching has been during a pandemic, I have loved every minute and would advise anyone thinking of a change of careers that you will be embraced and more importantly an asset to the profession and the students.
```

Comment:
```suggestion
As far as benefits are concerned, I am paid at a graduate level and have excellent holidays, which I get to spend with my family because we're all off together at the same time. In all, it was a great decision. Even though my whole training and subsequent teaching has been during a pandemic, I have loved every minute. I would advise anyone thinking about changing careers that you will be embraced and more importantly an asset to the profession and students.
```
Quite a long, complex sentence. Can we split up and simplify for readability?